apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -0,0 +1,823 @@
+                                                    );
+                                                } }
+                                            > OpenID Connect Scopes.
+                                            </Link>
+                                            You can map additional attributes under
+                                            <Link
+                                                external={ false }
+                                                onClick={ () => {
+                                                    history.push(
+                                                        AppConstants.getPaths()
+                                                            .get("ATTRIBUTE_MAPPINGS")
+                                                            .replace(":type", ClaimManagementConstants.OIDC)
+                                                    );
+                                                } }
+                                            > Attribute.
+                                            </Link>
+                                        </Trans>
+                                    </Hint>
+                                )
+                                : (
+                                    <Hint>
+                                        <Trans
+                                            i18nKey={
+                                                "console:develop.features.applications.edit.sections.attributes." +
+                                                "selection.attributeComponentHintAlt"
+                                            }
+                                        >
+                                        Manage the user attributes you want to share with this application.
+                                        You can add new attributes and mappings by navigating to
+                                            <Link
+                                                external={ false }
+                                                onClick={ () => {
+                                                    history.push(
+                                                        AppConstants.getPaths()
+                                                            .get("LOCAL_CLAIMS")
+                                                    );
+                                                } }
+                                            > Attribute.
+                                            </Link>
+                                        </Trans>
+                                    </Hint>
+                                )
+                            }
+                        </Grid.Column>
+                    </Grid.Row>
+                </>
+            )
+            : null
+    );
+};
+
+/**
+ * Default props for the application attribute selection component.
+ */
+AttributeSelectionOIDC.defaultProps = {
+    "data-testid": "application-attribute-selection"
```

Comment:
Can we give a different data-testid?
https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx#L1160

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3528/commits/acf93bbed6a21c13a9d22d84cc8294ea664ad8c4